challengers/Sandesh_Niraula/medium/day7-medium-task-4-markdown-to-html/index.html

```
@@ -0,0 +1,504 @@
+                <div class="guide-item">
+                    <h3>Code</h3>
+                    <code>`inline code`
+
+\`\`\`
+code block
+\`\`\`</code>
+                </div>
+
+                <div class="guide-item">
+                    <h3>Other</h3>
+                    <code">> Blockquote
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)